Assets/Tests/TestBulletPatterns.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Tests
+{
+    public class TestBulletPatterns
```

Comment:
Normally in C#, there's a decorator to mark a class as a test class. Does that apply to Unity?

Reply:
No, not to my knowledge. Unity does some injection stuff, so perhaps decorators may not be helpful here? I used: https://www.raywenderlich.com/9454-introduction-to-unity-unit-testing

as reference.